components/Gimme.vue

```diff
@@ -118,15 +127,21 @@ onMounted(() => {
     },
   }
 
-  let extentCommunities = communitiesWithinExtent()
-  if (extentCommunities.length > 0) {
+  if (!communitiesEnabled) {
     config.data = {
-      src: extentCommunities,
-      keys: ['name', 'alt_name'],
+      src: [],
     }
   } else {
-    config.data = {
-      src: [],
+    let extentCommunities = communitiesWithinExtent()
+    if (extentCommunities.length > 0) {
+      config.data = {
+        src: extentCommunities,
+        keys: ['name', 'alt_name'],
+      }
+    } else {
+      config.data = {
+        src: [],
+      }
```

Comment:
I believe you could simply change the else on line 134 to if (communitiesEnabled) and get rid of lines 130-133 to get the same result.